content/servicemesh/visualize.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Monitor & Visualize"
+date: 2018-11-13T22:42:01+09:00
+weight: 60
+draft: false
+---
+
+### Collecting new telemetry data
+
+Download a YAML file to hold configuration for the new metric and log stream that Istio will generate and collect automatically.
+
+```
+curl -L https://bit.ly/2PoBefv -o istio-telemetry.yaml
```

Comment:
is there complete url rather than using bit.ly link?

Reply:
replaced bitly to complete url